### ISSUE TITLE
Parameter Settings about COCO Detection

### ISSUE THREAD

**Issue by mayoycy** [facebookresearch/ConvNeXt-V2#13]
In the paper it was writen that "sweep layer-wise learning rate decay in {0.9, 0.95}, stochastic depth rate in {0.2, 0.3, 0.4, 0.5}". I'd like to know which of them are your recommended parameters with **ConvNeXt-V2 Base** Mask-RCNN in COCO task.

**Comment by ianpundar** [comment]
Have you successfully used the pretrained model?
